Formula/bash.rb

```diff
@@ -30,6 +30,7 @@ class Bash < Formula
       015 a517df2dda93b26d5cbf00effefea93e3a4ccd6652f152f4109170544ebfa05e
       016 ffd1d7a54a99fa7f5b1825e4f7e95d8c8876bc2ca151f150e751d429c650b06d
       017 4cf3b9fafb8a66d411dd5fc9120032533a4012df1dc6ee024c7833373e2ddc31
+      018 b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d
```

Comment:
I was about to submit this myself, and I think this should be:
```
018 7c314e375a105a6642e8ed44f3808b9def89d15f7492fe2029a21ba9c0de81d3
```

Reply:
yeah, you are right. :)